### PR TITLE
Add `rust-toolchain.toml` to raytracing example

### PR DESCRIPTION
**Pull request by jthemphill**
The raytracing example requires [a nightly build of Rust, and the source code to the Rust std library](https://rustwasm.github.io/docs/wasm-bindgen/examples/raytrace.html).

Running `build.sh` while your stable toolchain is active will result in an error:

```
error: the `-Z` flag is only accepted on the nightly channel of Cargo, but this is the `stable` channel
```

I tried changing `build.sh` to run `cargo +nightly`, but this broke CI.

My inclination here is to describe these dependencies with [a toolchain file](https://rust-lang.github.io/rustup/overrides.html#the-toolchain-file). It's one less build flag to forget.

### CODE REVIEW

File: examples/raytrace-parallel/rust-toolchain.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "nightly"
+components = ["rust-src"]
+targets = ["wasm32-unknown-unknown"]
```
Comment (daxpedda) [inline examples/raytrace-parallel/rust-toolchain.toml:4]:
```suggestion
targets = ["wasm32-unknown-unknown"]

```

**Review by daxpedda — CHANGES_REQUESTED** [review]
Thank you, this is great!

File: examples/raytrace-parallel/rust-toolchain.toml (hunk 1 above)
Comment (daxpedda) [inline examples/raytrace-parallel/rust-toolchain.toml:4]:
```suggestion
targets = ["wasm32-unknown-unknown"]
profile = "minimal"
```
Would this work as well?

**Review by daxpedda — CHANGES_REQUESTED** [review]
CI is failing because of the toolchain file.
I just took a quick look at the documentation to see if there is some flag to ignore it but couldn't really find anything. Would you mind exploring what to do here?

A quick solution that comes to mind here is to just use `+stable` to override the toolchain file.

File: examples/raytrace-parallel/rust-toolchain.toml (hunk 1 above)
Comment (jthemphill) [reply examples/raytrace-parallel/rust-toolchain.toml:4]: I suppose we can give it a try...

The error we're getting is

```
error: 'cargo-clippy' is not installed for the toolchain 'nightly-x86_64-unknown-linux-gnu'
```

And `profile = "minimal"`  avoids installing clippy.

I don't see how not installing clippy will help us with a "clippy not installed" error. Could someone change the CI action to run `cargo +stable clippy`, or to install nightly clippy before checking the examples that require a nightly build?

**Comment by jthemphill** [comment]
Okay, I do think the clippy CI run needs to either install a nightly version of clippy, or it needs to change the command from `cargo clippy` to `cargo +stable clippy` to specify that we use stable clippy even when a`rust-toolchain.toml` file calls for nightly.

File: examples/raytrace-parallel/rust-toolchain.toml (hunk 1 above)
Comment (daxpedda) [reply examples/raytrace-parallel/rust-toolchain.toml:4]: Ah no, these two problems are unrelated. I was just asking if `profile = "minimal"` is enough to run this example, that's all!

**Comment by daxpedda** [comment]
> `cargo +stable clippy`

I'm in favor of that.

**Review by daxpedda — APPROVED** [review]
Thanks.

**Pull request merged**